presets/workspace/models/deepseek/model.go

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+package deepseek
+
+import (
+	"time"
+
+	kaitov1alpha1 "github.com/kaito-project/kaito/api/v1alpha1"
+	"github.com/kaito-project/kaito/pkg/model"
+	"github.com/kaito-project/kaito/pkg/utils/plugin"
+	"github.com/kaito-project/kaito/pkg/workspace/inference"
+	"github.com/kaito-project/kaito/pkg/workspace/tuning"
+)
+
+func init() {
+	plugin.KaitoModelRegister.Register(&plugin.Registration{
+		Name:     PresetDeepSeekR1DistillLlama8BModel,
+		Instance: &deepseekA,
+	})
+	plugin.KaitoModelRegister.Register(&plugin.Registration{
+		Name:     PresetDeepSeekR1DistillQwen14BModel,
+		Instance: &deepseekB,
+	})
+}
+
+var (
+	PresetDeepSeekR1DistillLlama8BModel         = "deepseek-r1-distill-llama-8b"
+	PresetDeepSeekR1DistillQwen14BModel         = "deepseek-r1-distill-qwen-14b"
+
+	PresetDeepSeekTagMap = map[string]string{
+		"DeepSeekDistillLlama8B":          "0.0.1",
+		"DeepSeekDistillQwen14B":          "0.0.1",
+	}
+
+	baseCommandPresetDeepseekInference = "accelerate launch"
+	baseCommandPresetDeepseekTuning   = "cd /workspace/tfs/ && python3 metrics_server.py & accelerate launch"
+	deepseekLlama8bRunParams = map[string]string{
+		"torch_dtype":   "bfloat16",
+		"pipeline":      "text-generation",
+		"chat_template": "/workspace/chat_templates/deepseek-r1-distill-llama-8b.jinja",
```

Comment:
remove this line. tokenzier will automatically loaded it from tokenzier_config.json.

Reply:
ok i thought this, we have included jinja template for completeness